### PR TITLE
Streamline the build process

### PR DESCRIPTION
**Pull request by daPhipz**
This is a rather large PR, but I'll try to break it down here a little bit.

**The main goal: Streamline the build process.**
`*.mo` files are just "machine readable" binary versions of the corresponding `*.po` files. So there's no real reason to keep them in the tracked codebase - instead, this is a very good opportunity for automation!

**Changes in the files:**
I changed the lines in **`package.json`**, so that it now calls shell scripts in the `install` and `prepack` step. This allows to do maaany things in one step, without `package.json` ending up with one big line of code that is hard to read and debug.

In **`install.sh`**, I basically just took the existing line from `package.json` and broke it down with backslashes. This helps with readability and debuggability. And I added an error message for the case one of the steps fails.

**`prepack.sh`** is new. This is where the `*.mo` creation is done. As the filename suggests, it runs directly before the `pack` step.
I hope that the comments are somewhat helpful ;)
Basically, it looks for any `.po` file, iterates over them and creates the `.mo` counterpart in the correct location.
I also added a test if all necessary packages are installed in order to do this procedure - for possible new collaborators, to get a neat experience ;)

That's it for the moment - do not hesitate to ask questions or request changes! I make quite big changes here, so you have a point in checking everything that I do thoroughly ;)

### CODE REVIEW

**Comment by dallagi** [comment]
Hi @daPhipz , thanks for your contributions!

I like this PR, I only have a couple points I'd like to discuss:
1. Are you sure about having `prepack` and `pack` as separate steps? Why not a single `pack` step that builds the plugin, translations included?
2. Less importantly: personally I tend to prefer comments to be concise, especially when the code is quite clear by itself. For example, I'd remove the comment `# The error message gets only shown if one of the three commands linked with '&&' fails.`, and perhaps I'd change the first comments in the scripts from e.g., `Script to pack and install the extension locally, mostly for testing purposes` to e.g., just `Pack and install the extension locally.`

What do you think?

**Comment by daPhipz** [comment]
> Are you sure about having `prepack` and `pack` as separate steps? Why not a single `pack` step that builds the extension, translations included?

Yes, we could totally do this. It just sounded better in my head, since the locale generation happens before the actual packing and has nothing to do with it directly. So, personal preference, I guess :joy: Should I change it?

OK on the comments side!

**Comment by dallagi** [comment]
Yeah you're right, `pack` would be a weird name for a step that does more than just packing the extension.

But still, I don't see a good reason to have them as separate tasks...
Perhaps we could rename it to be more fitting to this use case? What about having a `build` task that both prepares the extension and builds the zip artifact?

**Comment by daPhipz** [comment]
You mean combining what is now `prepack` and `pack` and call it `build? Seems good, will do that!

**Comment by dallagi** [comment]
Yeah exactly. Again, thanks a lot for your work! 😄

**Comment by daPhipz** [comment]
No problem!
I like your extension very much and I am currently working on adding date and weekday support - it will look like this:
`Sun, 22th Nov - Half past six`

I have nothing PR-worthy yet, but little steps, I guess :joy:

**Comment by dallagi** [comment]
Great to hear :smile: 

BTW there'd be a small fix to be done (renaming `pack` to `build` in the install.sh script) but I'll do it myself after merging.
Thank you again!

**Comment by daPhipz** [comment]
Oh yeah, forgot that. Thank you!
